test/release/examples/primers/chplvis/chplvis.html

```diff
@@ -0,0 +1,312 @@
+directory is created, one can run chplvis as <em>chplvis E1</em> or
+simply <em>chplvis</em> and then opening the file <em>E1/E1-0</em>
+from the <em>file/open</em> menu.  The resulting display is:
+</p>
+<img src="E1.png" style="margin:10px; border: 2px solid black;">
+<p>
+(Note: This image is from an X11 display.  On OS-X, the menu bar
+will be on the normal menu bar at the top of the screen and will
+not show in the main window.)
+</p>
+<h4>chplvis Elements</h4>
+<ul>
+<li><em>Locale</em> is represented by a box.  The initial display
+draws the color of the locale to represent the number of tasks run
+at that locale.   For example 1, we can see that locale 0 has the
+most tasks and we expect that to be 13 since that is the maximum
+number of tasks as shown by the color reference at the top of the
+window.</li>
+<li><em>Communication links</em> are shown by lines between two
+locale boxes.  The color of the line adjacent to a locale box 
+represents the data being sent to that locale from the locale on
+the other end.   For example 1, the line between locale 0 and
+locale 1 is colored red next to locale 0.  This means that there
+is a lot of communications <em>into locale 0</em> from locale 1.
+The blue line next to locale 1 means that there is little 
+communication <em>into locale 1</em> from locale 0.   <em>Note:</em>
+If two locales do not communicate, no line is drawn between them.
+If communication is only one way, the communication color for
+<em>no communication</em> is gray.</li>
+<li><em>Data menu</em> controls what data is used for display
+colors.   This initial data is for number of tasks for locales
+and number of communications for the communication links.
+For locales, one can select number of tasks, CPU time at the locale
+or clock time at the locale.  Clock time is normally very close to
+equal across all locales.  For the communication links, one can
+select number of communications or size of data sent.</li>
+</ul>
+<h4>Display Interaction</h4>
+<p>
+Clicking on elements of the display will bring up more information.
+Clicking on a locale will open a window for that locale showing
+information for that locale.  For example, in example 1, clicking on
+locale 0 will produce a window that looks like:
+</p>
+<img src="E1-L0.png" style="margin:10px; border: 2px solid black;">
+<p>
+(Note: There is overhead generated in tasks, CPU time and clock
+time for the Visual Debug function calls.  <em>chplvis</em> removes
+from the task count the overhead tasks but it can not remove the
+CPU and clock time overhead.)
+</p>
+<p>
+Clicking on the red part of the line between locale 0 and locale 1
+will produce a window that looks like:
+</p>
+<img src="E1-C1.0.png" style="margin:10px; border: 2px solid black;">
+<p>
+It is important to notice the direction of the "arrow" in the header
+for the windows.   This is for communication from locale 1 to locale 0.
+The total communication was 12.  It is further broken out into three
```

Comment:
I'd like this to say "The total number of communication calls was 12", or something along those lines